### DISCUSSION TITLE
How to use tabs and launch for splitting _without_ cloning ssh session?

### DISCUSSION THREAD

**Discussion by hedefalk** [kovidgoyal/kitty#6015] [Q&A] [answered]
I've been using this config for a couple of years:

```
map cmd+t new_tab_with_cwd
map cmd+shift+d launch --location=hsplit --cwd=current
map cmd+d launch --location=vsplit --cwd=current
map cmd+w close_window
```

These are the only shortcuts I use to create new tabs and such. Before, these have never kept/started new ssh connections, but just worked with current directory locally. 

Typical scenario for me:

* Start kitty
* ssh into remote server
* hit cmd+shift+d to open another terminal on my _local_ machine to do some local work.

Now this new terminal is automatically connected to the remote over ssh. Super cool, but I don't even now how to open a new kitty window to my local machine if I have an ssh connection open. The only way I'm able to do that is by opening another os window (cmd-n in my case) or exiting my first ssh session…

Is there a way I can change my shortcuts so they behave like before?

**Comment by hedefalk** [comment]
I'm also using 

```
 alias ssh="kitty +kitten ssh"
```

**Comment by kovidgoyal** [comment]
--cdw=current will clone your current directory which with a remote
connection means cloning the remote connection. If you want to open a
window without cloning the cwd create a mapping that does not use it.

**Reply by hedefalk** [reply]
Yeah, I'm wondering if there is any way to keep cwd iff local session, but not do the smart ssh thing.

Cloning the remote connection is super useful and I'm gonna use that too, and maybe I've been misusing launch for my usecase, but basically I need some way of just opening vertical/horizontal/tabs with cwd but _not_ ssh…? 

Tried last_reported but same behaviour…

**Comment by kovidgoyal** [comment, marked as answer]
No, there's no such facility. I dont really get the use case for it. You
want to be able to clone local environments but not remote ones? Why?
If you have only one map that clones local but not remote environments,
you will *never* be able to clone remote environments. Far better to
have two maps, one that clones both and one that clones neither.

**Reply by hedefalk** [reply]
It's just ergonomics and something that I'm used to. I've always been cloning cwd as a default - I have zero need launching into HOME.

But with my current use of launch with --cwd=current, I can't reach my own computer if I already have an ssh up. I guess this comes up because launch is my _only_ way of starting new terminals.

My current workaround is just to not use ssh kitty in the normal case, and if I feel the need to clone ssh sessions I'll have to go back and use a separate ssh alias in the first instance…
